src/screens/settings/SettingsScreen.tsx

```diff
@@ -25,6 +30,67 @@ const SettingsScreen = ({ navigation }: any) => {
   const [password, setPassword] = useState('');
   const [language, setLanguage] = useState(''); // string type in Firebase
   const [currentPassword, setCurrentPassword] = useState('');
+  const db = firebase.firestore();
+  const clientCollection = db.collection('clients');
+  const { userLanguageChange } = React.useContext(LanguageContext);
+  const { state } = React.useContext(ClientContext);
+
+  const updateLanguage = async (lang: string) => {
```

Comment:
I moved these functions to database/auth to separate code we render from backend queries so could you apply this refactor to those pre-existing functions?

Reply:
Ty--and yup, I'll refractor the pre-existing functions.